ui/page/components/components.go

```diff
@@ -165,6 +166,105 @@ func TransactionTitleIcon(l *load.Load, wal *dcrlibwallet.Wallet, tx *dcrlibwall
 	return &txStatus
 }
 
+func DurationAgo(timestamp int64) string {
+	//Convert timestamp to date in string format (yyyy:mm:dd hr:m:s +0000 UTC)
+	currentTimestamp := time.Now().UTC().String()
+	txnTimestamp := time.Unix(timestamp, 0).UTC().String()
+	//Split the date so we can sepparate into date and time for current time and time of txn
+	currentTimeSplit := strings.Split(currentTimestamp, " ")
+	txnTimeSplit := strings.Split(txnTimestamp, " ")
+	//Split current date and time, and  txn date and time then store in variables
```

Comment:
```suggestion
	// Convert timestamp to date in string format (yyyy:mm:dd hr:m:s +0000 UTC)
	currentTimestamp := time.Now().UTC().String()
	txnTimestamp := time.Unix(timestamp, 0).UTC().String()
	// Split the date so we can sepparate into date and time for current time and time of txn
	currentTimeSplit := strings.Split(currentTimestamp, " ")
	txnTimeSplit := strings.Split(txnTimestamp, " ")
	// Split current date and time, and  txn date and time then store in variables
```
Leave a space before typing the comments